src/kolmogorov_smirnov.jl

```diff
@@ -54,6 +54,15 @@ immutable ExactOneSampleKSTest <: ExactKSTest
     δn::Float64 # supremum of the negative CDF differences
 end
 
+"""
+    ExactOneSampleKSTest{T<:Real}(x::AbstractVector{T}, d::UnivariateDistribution)
+
+Perform a one sample Kolmogorov–Smirnov-test of the null hypothesis that the data in vector
```

Comment:
"Kolmogorov–Smirnov test". Same below.

Would also make sense to mention "exact".